using_images/other_images/jenkins.adoc

```diff
@@ -52,9 +52,24 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
 
 === Initializing Jenkins
 
+ifdef::openshift-origin[]
+There are two methods for managing authentication with the {product-title} Jenkins images when they are initially started.
+
+* Default Jenkins authentication
+
+* {product-title} OAuth authentication provided by the OpenShift Login plugin.
+
+==== Default Jenkins authentication
+
+Default Jenkins authentication will be used when the {product-title} Jenkins images are first launched outside of an {product-title} pod,
+or when the `*OPENSHIFT_ENABLE_OAUTH*` environment variable is either missing or set to `false` when running inside a pod.
```

Comment:
doesn't this boil down to "jenkins auth will be used by default, openshift oauth will be used when the image is running in a pod on an openshift cluster and the openshift_enable_oauth env variable is set to a value other than 'false' "?

also this implies we can't enable openshift oauth when running outside openshift...is that true? if so why?